src/platform/posix/include/platform/lib/memory.h

```diff
@@ -39,11 +39,6 @@ static inline void *platform_shared_get(void *ptr, int bytes)
 	return ptr;
 }
 
-#define SRAM_BANK_SIZE 0x10000
-#define EBB_BANKS_IN_SEGMENT 32
-#define PLATFORM_HPSRAM_EBB_COUNT 32
-#define PLATFORM_LPSRAM_EBB_COUNT 1
-
```

Comment:
This is no longer needed as these definitions are only used in the Intel specific basefw code (so not needed to build for posix).